tests/nosetests/pyanaconda_tests/module_storage_test.py

```diff
@@ -99,6 +99,19 @@ def _test_dbus_partitioning(self, publisher, expected_method):
         self.assertIsInstance(obj.implementation, PartitioningModule)
         self.assertEqual(partitioning_module, obj.implementation)
 
+    def _apply_partitioning_when_created(self):
+        """Apply partitioning when it is created."""
```

Comment:
What's it? Could you please rephrase this?

Reply:
Fixed.